test-jw/src/videoConfigs.js

```diff
@@ -6608,5 +6608,6 @@ export const ARTICLE_VIDEO_DETAILS = {
                 "fcc4038c": 0.7692186749171301
             }
         }
-    }
+    },
+    "tier3Mapping": false,
 }
```

Comment:
Missing newline at the end of the file.

Reply:
Fixed in fcc33538c5143985c81ccbc7bd821c5193df6c63